tests/unit/h/presenters/mention_json_test.py

```diff
@@ -29,7 +33,10 @@ def test_as_dict_with_different_username(self, user, annotation, pyramid_request
 
     @pytest.fixture
     def user(self, factories):
-        return factories.User.build()
+        return factories.User.build(
+            description="user description",
+            activation_date=factory.Faker("date_time_this_decade"),
```

Comment:
Should this be `created`?

```suggestion
            created=factory.Faker("date_time_this_decade"),
```

Reply:
No, on the model itself there's no `created`
Isn't that what we [went over above](https://github.com/hypothesis/h/pull/9349#discussion_r1954112584), exposing `activation_date` as `created` to hide the details?

Reply:
Oh sorry! You are absolutely right.